### DISCUSSION TITLE
How to watch all the duplicate groups ?

### DISCUSSION THREAD

**Discussion by imohitkr** [immich-app/immich#11632] [Q&A] [answered]
I was reviewing the duplicates on Web interface and it shows I have ~2000 duplicates. How to see all the individual groups at once ?

**Comment by mertalev** [comment, marked as answer]
It isn't possible to view them all at once right now. It's something to improve on for sure.

**Reply by imohitkr** [reply]
Okay, thanks hopefully we get this soon.
